packages/edit-post/src/components/options-modal/index.js

```diff
@@ -65,6 +65,20 @@ export function OptionsModal( { isModalActive, isViewable, closeModal } ) {
 					label={ __( 'Contain text cursor inside active block' ) }
 				/>
 			</Section>
+			<Section title={ __( 'Writing Mode' ) }>
+				<EnableFeature
+					featureName="themeStyles"
+					label={ __( 'Theme Styles' ) }
+				/>
+				<EnableFeature
+					featureName="reducedUI"
+					label={ __( 'Compact UI' ) }
+				/>
+				<EnableFeature
+					featureName="focusMode"
+					label={ __( 'Spotlight' ) }
+				/>
+			</Section>
```

Comment:
That's a lot of options. Why not merge with spotlight?

Reply:
Worth keeping in mind https://wordpress.org/about/philosophy/#decisions

Reply:
This is one place where we can't make decision in behalf of the user IMO. Someone might want the spotlight effect combined with the top toolbar while someone else might want to remove the theme styles, use an inline toolbar and

Reply:
Let's see if we can get to a better "preferences" UI next: #24965